assets/js/app.js

```diff
@@ -23,7 +23,13 @@ import { LiveSocket } from 'phoenix_live_view'
 import topbar from '../vendor/topbar'
 import AsciinemaHook from './hooks/asciinema_hook'
 import CopyHook from './hooks/copy_hook'
+document.addEventListener('DOMContentLoaded', () => {
+  const toolInstallationElement = document.getElementById('tool-installation')
 
+  if (toolInstallationElement) {
+    toolInstallationElement.scrollRight = toolInstallationElement.scrollWidth
```

Comment:
What's the purpose of this? Is it necessary?

Reply:
Oghogho, had mentioned that the command is too long on mobile screens so the users can't actually see the copy icon, so we needed a way to scroll the command on say mount, so that they can see the copy icon.

Reply:
We can use hooks actually.